pyoozie/tags.py

```diff
@@ -16,6 +19,9 @@
 ALLOWABLE_NAME_CHARS = set(string.ascii_letters + string.punctuation + string.digits + ' ')
 ONE_HUNDRED_YEARS = 100 * 365.24
 
+PropertyValuesType = typing.Optional[typing.Dict[typing.Text, typing.Any]]
```

Comment:
Where does `typing.Text` come from? It's not in the docs for Python 3.5.2, according to Dash.